actions/executions_create.py

```diff
@@ -0,0 +1,45 @@
+from lib.action import St2BaseAction
+from st2client.models.action import Execution
+from st2client.commands.resource import ResourceNotFoundError
+
+__all__ = [
+    'St2ExecutionsCreateAction'
+]
+
+
+def format_result(item):
+    if not item:
+        return None
+
+    return item.to_dict()
+
+
+class St2ExecutionsCreateAction(St2BaseAction):
+    def run(self, action, parameters=None):
+        parameters = parameters or {}
+
+        action_resource = self.client.managers['Action'].get_by_ref_or_id(action)
+        if not action_resource:
+            raise ResourceNotFoundError('Action "%s" cannot be found.'
+                                        % action)
+        action_ref = '.'.join([action_resource.pack, action_resource.name])
+
+        execution = Execution()
+        execution.action = action_ref
+        execution.parameters = parameters
```

Comment:
To avoid mixing action parameters and additional options supported by the schedule execution API, I would make ``user``, ``delay``, ``trace_id`` and ``trace_tag`` separate action parameters which are not mixed with action parameters.

Reply:
Done.